clar2wasm/src/standard/standard.wat

```diff
@@ -1541,23 +1623,29 @@
             )
         )
         i64.store offset=288
-
+        
         (local.get $res_len)
     )
 
     (func $block64 (param $data i32)
         (local $origin i32)
         (local $i i32) (local $tmp i32)
 
+        ;; Frequently accessing global variable can cause performance issues, that's why a local one is used
```

Comment:
We don't really want to modify stack-pointer anyway, anything that will be stored there is temporary.

Reply:
We're not modifying `$origin` either.

Reply:
yeah, I was adding more info to your comment.

Reply:
Added